### DISCUSSION TITLE
passwall和 passwall2   访问控制→接口控制，无法正常工作，请问什么情况？

### DISCUSSION THREAD

**Discussion by Zren215** [Openwrt-Passwall/openwrt-passwall2#681] [Q&A]
访问控制   选择接口控制，  我自己设置了好几个wifi信号，使用接口控制，设备走指定的wifi接口控制，无法实现，走设置好的节点线路。    但是  可以通过设备的mac地址，进行访问控制，走指定的节点，很完美实现。  
 请问，怎么回事？？是不是接口控制   有bug？？？麻烦请改正。 谢谢

**Comment by lwb1978** [comment]
@WROIATE

**Comment by WROIATE** [comment]
配置文件能发下吗，或者配置截图
记得去掉敏感信息
